packages/venia-ui/lib/fragments/productDetails.graphql

```diff
@@ -1,5 +1,11 @@
 fragment ProductDetails on ProductInterface {
     __typename
+    categories {
```

Comment:
As an FYI we should add a comment in here at the top saying that updates to this fragment should be mirrored in `getProductDetail` and vice versa.